sci-visualization/mayavi/mayavi-4.4.0.ebuild

```diff
@@ -0,0 +1,77 @@
+# Copyright 1999-2015 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Header: Exp $
+
+EAPI=5
+
+PYTHON_COMPAT=( python2_7 )
+
+inherit distutils-r1 virtualx
+
+DESCRIPTION="Enthought Tool Suite: Scientific data 3-dimensional visualizer"
+HOMEPAGE="
+	http://code.enthought.com/projects/mayavi/
+	http://pypi.python.org/pypi/mayavi/"
+SRC_URI="https://github.com/enthought/${PN}/archive/${PV}.tar.gz"
+
+LICENSE="BSD"
+SLOT="2"
+KEYWORDS="~amd64 ~x86 ~amd64-linux ~x86-linux"
+IUSE="doc examples test"
+
+RDEPEND="
+	>=dev-python/apptools-4[${PYTHON_USEDEP}]
+	dev-python/configobj[${PYTHON_USEDEP}]
+	>=dev-python/envisage-4[${PYTHON_USEDEP}]
+	dev-python/ipython[${PYTHON_USEDEP}]
+	dev-python/numpy[${PYTHON_USEDEP}]
+	dev-python/pyface[${PYTHON_USEDEP}]
+	>=dev-python/traitsui-4[${PYTHON_USEDEP}]
+	dev-python/wxpython[${PYTHON_USEDEP}]"
+CDEPEND="sci-libs/vtk[python]"
+DEPEND="
+	${CDEPEND}
+	dev-python/setuptools[${PYTHON_USEDEP}]
+	doc? ( dev-python/sphinx[${PYTHON_USEDEP}] )
+	test? (
+		${RDEPEND}
+		dev-python/nose[${PYTHON_USEDEP}]
+		dev-python/wxpython[opengl]
```

Comment:
Eh? It seems to work with 2.8, but I'm not sure if it cannot work with 2.9 or 3.0. Right now I'm also too short of time to test this.

Regarding `PYTHON_USEDEP`, would `dev-python/wxpython[opengl,${PYTHON_USEDEP}] be the right answer to this riddle?